java/openjdk10/Portfile

```diff
@@ -31,22 +30,21 @@ homepage         https://jdk.java.net/${openjdk_version}/
 
 if {${subport} eq ${name}} {
     # openjdk10
-    master_sites     https://download.java.net/java/GA/jdk10/${version}/19aef61b38124481863b1413dce1855f/13/
-    distname         openjdk-${version}_osx-x64_bin
+    master_sites     https://download.java.net/java/GA/jdk${openjdk_version}/${version}/19aef61b38124481863b1413dce1855f/13/
 
     checksums        rmd160  d29498411adc487bf8191adbc4276c72602022cf \
                      sha256  77ea7675ee29b85aa7df138014790f91047bfdafbc997cb41a1030a0417356d7 \
                      size    200916897
 } else {
     # openjdk11
-    master_sites     https://download.java.net/java/GA/jdk${version}/${build}/GPL/
-    distname         openjdk-${version}+${build}_osx-x64_bin
+    master_sites     https://download.java.net/java/ga/jdk${openjdk_version}/
```

Comment:
This link uses "ga" and above for openjdk10 it is "GA" (all caps). I am assuming both should be the same?

It's too bad the `master_sites` line cannot be same for both.

Reply:
I don’t know, these are the links as published on https://jdk.java.net/10/ and https://jdk.java.net/11/ respectively.

Reply:
Yes strange.